### PR TITLE
[Bug] Change visual helper text to "Highlight instances on target page"

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

Change "Visual Helper" text to "Highlight instances on target page".

#### Pull request checklist

- [x] Addresses an existing issue: Fixes  WI 1457433
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [x] Added screenshots/GIFs for UI changes.

#### Screenshots:

image

### CODE REVIEW

File: src/DetailsView/components/base-visual-helper-toggle.tsx [hunk 1]
```diff
@@ -18,13 +17,13 @@ export abstract class BaseVisualHelperToggle extends React.Component<VisualHelpe
 
         return (
             <div className="visual-helper">
-                <div className="visual-helper-text">Visual helper</div>
+                <div className="visual-helper-text">Highlight instances on target page</div>
```
Comment (waabid) [inline src/DetailsView/components/base-visual-helper-toggle.tsx]: I'd prefer if we are using this in multiple places to just keep it as a constant somewhere that we can access.

Comment (Shobhit1) [reply src/DetailsView/components/base-visual-helper-toggle.tsx]: My first intention was to just make this change without making more substantial code changes. But I guess it makes sense for me to make this change.

Comment (waabid) [reply src/DetailsView/components/base-visual-helper-toggle.tsx]: It's just easier for the test changes in the future, I think. Thanks! 😄

Comment (Shobhit1) [reply src/DetailsView/components/base-visual-helper-toggle.tsx]: Let me know if the current change feels better to you.

**Pull request merged**